distributions/common/k8s/worker-deployment.yaml

```diff
@@ -16,15 +16,15 @@
 # -----------------------------------------------------------------------
 # Kubernetes deployment for workers. Workers each process one job, and
 # terminate themselves at the end of it. The "Always" restart policy in
-# this configuration is necessary to constantly refill the worker pool
+# this configuration is necessary to constantly refill the transfer pool
 # as workers complete jobs and terminate.
 #
 apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
   labels:
     app: portability
-  name: portability-worker
+  name: portability-transfer
```

Comment:
Good question, we might as well change worker->transfer here: https://github.com/google/data-transfer-project/blob/master/distributions/demo-google-deployment/bin/build_docker_image.sh 

I think that file will be eliminated once we build containers using the gradle plugin, but for now it would be good to keep it all consistent.

Would be good to name consistently here too: https://github.com/google/data-transfer-project/blob/master/distributions/demo-google-deployment/bin/setup_gke_environment.sh

Reply:
Done. PTAL.

Reply:
LGTM